### ISSUE TITLE
ebiten: New debug mode to unmerge all the draw calls

### ISSUE THREAD

**Issue by hajimehoshi** [hajimehoshi/ebiten#1371]
CC @mariosnikolaou

**Comment by hajimehoshi** [comment]
Sorry but I lost the context... I was wondering why we need this 🤔

**Comment by hajimehoshi** [comment]
Slack Log: https://gophers.slack.com/archives/C19491JS2/p1601342084117900

**Comment by blizzy78** [comment]
Right, I thought it would be quite useful to have a way to attach debugging info (log messages) to single draw calls or batches of draw calls. As in, when I call something like `screen.DrawImage`, I could somehow attach a log message "draw Gopher sprite" to it, and see that in the `ebitendebug` log.

I think it would be especially useful if Ebiten is still allowed to merge/batch draw calls because then you could see which of the program's actions go into an actual GPU draw call.

**Comment by hajimehoshi** [comment]
Sure, probably unmerging all the commands is not what we want. Attaching messages to a draw command and seeing them at the ebitendebug logs seems what we want.

**Comment by hajimehoshi** [comment]
Hmm, it is not obvious how to treat a comment between merged commands. We need more discussion and cannot make it for the next release unfortunately.
